namematch/predict.py

```diff
@@ -98,14 +98,16 @@ def get_potential_edges(self, dr_file, match_models, model_info,
             params (Parameters obj):  contains parameter values (i.e. use_uncovered_phats)
         '''
         try:
-            thread = dr_file.split('/')[-1].replace('data_rows_', '').replace('.parquet', '')
+            import pathlib
+            p = pathlib.Path(dr_file)
+            thread = p.parts[-1].replace('data_rows_', '').replace('.parquet', '')
+            #thread = dr_file.split('/').replace('data_rows_', '').replace('.parquet', '')
```

Comment:
We can just remove the commented out line.

Reply:
done